docs/getting-started.md

```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
   * Your iGPU
   * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
   * **Your boardname**
+       * For example, the boardname for Asus Chromebook C302CA is Cave
   * If you don't know where to find this info, take a look at the [System Info Page.](system-info.html)
 *  **A basic knowledge of command lines and how to use a terminal/command prompt.**
-  * We can't help you if you don't know how to cd to a directory or delete a file.
+  * We cannot help you if you do not know how to go to a directory or delete a file from a terminal.
 * A minimum of an 8GB USB drive or SD Card for creating a bootable OS installer.
+* A Chromebook running Ventoy (read [Ventoy](ventoy.html), or another computer running Balena Etcher, or Rufus (read [Making a Bootable USB](bootableusb.html)
```

Comment:
A chromebook is already a prerequisite

Reply:
its good to be explicit, since some people flash without having a bootable flash drive